nevergrad/benchmark/plotting.py

```diff
@@ -552,6 +571,28 @@ def make_data(df: pd.DataFrame, normalized_loss: bool = False) -> tp.Dict[str, t
                 optim_vals[optim]["pseudotime"] = np.array(means.loc[optim, "pseudotime"])
         return optim_vals
 
+    @staticmethod
+    def save_txt(output_filepath: tp.PathLike, optim_vals: tp.Dict[str, tp.Dict[str, np.ndarray]]) -> None:
+        """Saves a list of best performances.
+
+        output_filepath: Path or str
+            path where the figure must be saved
+        optim_vals: dict
+            dict of losses obtained by a given optimizer.
+        """
+        best_performance: tp.Dict[int, tp.Any] = defaultdict(lambda: (float("inf"), "none"))
+        for optim in optim_vals.keys():
+            for i, l in zip(optim_vals[optim]["budget"], optim_vals[optim]["loss"]):
+                if l < best_performance[i][0]:
+                    best_performance[i] = (l, optim)
+
+        with open(output_filepath, "w") as f:
```

Comment:
```suggestion
        with output_filepath.open("w") as f:
```

Reply:
Does not work because << nevergrad/benchmark/plotting.py:589: error: Item "str" of "Union[str, Path]" has no attribute "open"
 >>. What is the goal of this proposed change ?